sdks/python/apache_beam/dataframe/io.py

```diff
@@ -58,6 +58,17 @@
 _DEFAULT_BYTES_CHUNKSIZE = 1 << 20
 
 
+@frame_base.with_docs_from(pd)
+def read_gbq(table, *args, **kwargs):
+  """If you would like to use the 'DIRECT_READ' method ins ReadFromBigQuery,
+    please set use_bq_storage_api to True.
+    Otherwise, if you would like to use the 'EXPORT' method, please set
+    use_bq_storage_api to False, or leave it unspecified."""
```

Comment:
I could be swayed, but I think it would be preferable not to document how this is working under the hood (e.g. that it relies on `ReadFromBigQuery`), and instead just present the user-facing API.

Reply:
Hm, I disagree. I'm assuming that users that want to ReadFromBQ might already be use to the way the ReadFromBQ PTransform works, and thus it would be useful to let them know how to utilize 'DIRECT_READ' and 'EXPORT' via the ReadGbq PTransform.

What are the benefits of not mentioning 'DIRECT_READ' and 'EXPORT'?

Reply:
From the perspective of the users of this API, the mode is fully controlled by the `use_bqstorage_api` parameter. It just confuses things to reveal that it maps to mode={DIRECT_READ, EXPORT} under the hood.

If we want to reference the mode in the docstring, we should just have a `mode` parameter, and not bother with a `use_bqstorage_api` parameter. To be clear, I think that's an acceptable alternative, but I'd like to pick one or the other.

I have a slight preference to avoid using Beam terminology though - a goal of the DataFrame API is to provide an API that feels natural to developers who are already familiar with pandas, and not familiar with Beam.

Reply:
ah ok, thanks for sharing, I agree with your perspective on this and will use the pandas terminology.